README.md

```diff
@@ -37,13 +38,22 @@ CASA participants are structured into three groups:
 
 ### **CASA Meetings**
 
-All CASA stakeholders are invited to join a meeting every six weeks (attendance optional), for the purpose of:
+All CASA stakeholders are invited to join a meeting every two weeks, for the purpose of:
 
-- Sharing progress in individual working groups
-- Proposing the creation of new working groups
-- Evaluating CAIPs
+1. Evaluating CAIPs, registries, and related publications
+2. Sharing progress in individual working groups
+3. Proposing the creation of new working groups
 
-Since CASA meetings are primarily intended to sharing the working groups' progress, at least one member from each active working group is expected to attend.
+Since CASA meetings are primarily intended to sharing the working groups' progress, at least one member from each active working group is expected to attend, or entire working groups if the WG has topics agenda'd for a given meeting.
```

Comment:
```suggestion
Since CASA meetings are primarily intended to sharing the working groups progress, at least one member from each active working group is expected to attend, or entire working groups if the WG has topics agenda'd for a given meeting.
```